api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -351,41 +360,30 @@ public void testMe_userBeforeSuccessCloudProjectConflict() throws Exception {
     createUser();
     when(fireCloudService.isRequesterEnabledInFirecloud()).thenReturn(true);
 
-    String projectName = BILLING_PROJECT_PREFIX + user.getUserId();
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName);
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-1");
+    ApiException conflict = new ApiException(HttpStatus.CONFLICT.value(), "conflict");
+    doThrow(conflict)
+        .doThrow(conflict)
+        .doNothing()
+        .when(fireCloudService).createAllOfUsBillingProject(anyString());
 
     Profile profile = cloudProfileController.getMe().getBody();
 
     // When a conflict occurs in dev, log the exception but continue.
     assertProfile(profile, PRIMARY_EMAIL, CONTACT_EMAIL, FAMILY_NAME, GIVEN_NAME,
-        DataAccessLevel.UNREGISTERED, TIMESTAMP, projectName + "-2",
-        true, null);
+        DataAccessLevel.UNREGISTERED, TIMESTAMP, true, null);
+    assertThat(profile.getFreeTierBillingProjectName()).endsWith("-2");
```

Comment:
Will this work? I'm a bit concerned by the use of `anyString()` above.

Reply:
The test passes, see above comment.